MAINTAINER.md

```diff
@@ -40,6 +40,10 @@ The status `status/4-merge-in-progress` is only for the bot.
 If the bot is not able to perform the merge, the label `bot/need-human-merge` is added.  
 In this case you must solve conflicts/CI/... and after you only need to remove `bot/need-human-merge`.
 
+Maintainers can add `bot/no-merge` on a PR if want (temporarily) prevent the bot from merging.
+
+`bot/light-review` can be used to decrease required LGTM from 3 to 1.
```

Comment:
Do we also want to describe what kind of PRs are eligible for light reviews? (AFAIK, it's vendor updates from previously reviewed PRs and merges into master.)